contributing/content-style.md

```diff
@@ -316,7 +316,7 @@ Some of the rules are enforced with [Vale](https://docs.errata.ai/vale/about), a
 
 Because writing style is subjective and no checking tool is perfect,
 the rules are mostly set to warnings rather than errors.
-So checks note when something might be wrong,
+So check notes when something might be wrong,
```

Comment:
:lady_beetle: It was actually correct originally (before this change). But I can see how it would be confusing. The issue is that it uses the GitHub term `check` without really introducing it.

Perhaps something like this would help?
```suggestion
So automated checks show when something might be wrong,
```